content/en/tutorials/upgrading_eks.md

```diff
@@ -0,0 +1,81 @@
+which should cover Opta-managed EKS setups.
+
+# Steps
+As Kubernetes always [maintains 1 minor version compatibility backwards or forwards](https://kubernetes.io/releases/version-skew-policy/),
+all Kubernetes upgrades should be done one minor version at a time. EKS actually enforces this process. This
+means that if, for example, one wishes to go from version 1.18 to 1.21, they will need to do the update of
+1.18 -> 1.19, then 1.19 -> 1.20 and finally 1.20 -> 1.21. You can often do these repetitions in quick succession,
+but please read the [Breaking Changes](#breaking-changes) section carefully as some upgrades may require
+extra steps.
+
+To begin the upgrade, go to your AWS console UI, to the EKS section, and to the details of the cluster in question
+(make sure that you are in the correct region or otherwise you will not see the cluster's details). You should see a
+screeen like the following:
+
+<a href="/images/eks_upgrade_1.png" target="_blank">
+  <img src="/images/eks_upgrade_1.png" align="center"/>
+</a>
+
+## Step 1: Upgrade the Control Plane
+We will begin by updating the [Kubernetes control plane](https://kubernetes.io/docs/concepts/overview/components/#control-plane-components).
+
+To begin, simply click on the `Update Now` button which should be on your screen. It will lead you to a pop-up like so:
+
+<a href="/images/eks_upgrade_2.png" target="_blank">
+  <img src="/images/eks_upgrade_2.png" align="center"/>
+</a>
+
+**THIS IS IMPORTANT**: The control plane not only manages the container orchestration, but also the API through which users interact, meaning that
+no new deployments or `kubectl` access should be attempted while this step is being done (~20 minutes) Please schedule
+the upgrade time accordingly.
+
+Click `Update`, and wait until the update is complete.
+
+## Step 2: Upgrade the Nodes
+Once the control plane is upgraded, you will need to upgrade the individual nodes running your workloads. As Opta uses
+managed node groups (and by default installs at least 1), one can mass-upgrade the nodes by upgrading the managed node group.
+You can do this by going to the `Configuration` tab of your EKS cluster and clicking on the `Compute` section. You should
+see your managed node groups listed like so:
+
+<a href="/images/eks_upgrade_3.png" target="_blank">
+  <img src="/images/eks_upgrade_3.png" align="center"/>
+</a>
+
+If Step 1 was completed successfully, you should see an `Update now` link like in the example above. Click there to
+begin configuring the K8s upgrade. It will lead you to a pop-up like so:
+
+<a href="/images/eks_upgrade_4.png" target="_blank">
+  <img src="/images/eks_upgrade_4.png" align="center"/>
+</a>
+
+**THIS IS IMPORTANT**: Because all the nodes will be replaced, there will be a minute or two of downtime if you did not
+set the nginx ingress to high availability on your `k8s-base` instance, and all your currently running nodes containers
```

Comment:
add a link to /reference/aws/environment_modules/aws-k8s-base/ for user who don't know about this setting